app/src/App.js

```diff
@@ -60,21 +47,17 @@ const App = React.memo(() => {
               justify-content: center;
             `}
           >
-            <NoTokens onNewToken={requests.addToken} isSyncing={isSyncing} />
+            <NoTokens onNewToken={requests.updateTokens} isSyncing={isSyncing} />
           </div>
         )
       )}
       <SidePanel
-        title={getModeTag(mode)}
```

Comment:
Why are we evaluating in line instead of the previous getModeTag?

Reply:
Because now we have just 2 states instead of 3 so i think it's not necessary the function call and can be done inline. What do you think?

Reply:
oh ok,  i am okay with that!